.github/workflows/ci-formatting-and-dependencies.yml

```diff
@@ -34,4 +34,4 @@ jobs:
       - name: Test pip packages
         run: |
           cd $GITHUB_WORKSPACE
-          ./test_pip_packages.sh
\ No newline at end of file
+          . ./test_pip_packages.sh
```

Comment:
Yeah, I just got this at about the same time. The file is missing executable flags.